AllJoyn/Platform/Bridge/BridgeNET/BridgeNET.vcxproj

```diff
@@ -211,7 +212,9 @@
     <ClInclude Include="Stdafx.h" />
   </ItemGroup>
   <ItemGroup>
-    <ClCompile Include="AssemblyInfo.cpp" />
+    <ClCompile Include="AssemblyInfo.cpp">
+      <RuntimeLibrary Condition="'$(Configuration)|$(Platform)'=='Release|x64'">MultiThreadedDLL</RuntimeLibrary>
```

Comment:
#Resolved

---
In reply to: [75174478](https://github.com/tjaffri/msiot-samples/pull/9#discussion_r75174478) [](ancestors = 75174478)